common/output-model.cpp

```diff
@@ -1006,7 +1011,10 @@ void output_model::run_command(std::string command, device_models_list & device_
     }
     catch(const std::exception& ex)
     {
-        add_log( RS2_LOG_SEVERITY_ERROR, __FILE__, __LINE__, ex.what() );
+        std::string foo = rsutils::string::from() << ex.what();
```

Comment:
foo? :)
Please change to a meaningful name

Reply:
oops :)
fixed